cmd/vela-terraform/main.go

```diff
@@ -292,13 +294,19 @@ func run(c *cli.Context) error {
 		"registry": "https://hub.docker.com/r/target/vela-terraform",
 	}).Info("Vela Terraform Plugin")
 
-	// capture custom terraform version requested
-	version := c.String("terraform.version")
+	// capture custom terraform tfVersion requested
+	tfVersion := c.String("terraform.version")
+
+	tfSemVersion, err := semver.NewVersion(tfVersion)
```

Comment:
Need to handle the scenario where tfVersion is `""` since the user opted to utilize the pre installed terrafrom binary.

Reply:
we could shove https://github.com/go-vela/vela-terraform/blob/a00b059b732a3768b86de12364912b44bf5e253f/cmd/vela-terraform/main.go#L300-L304 into https://github.com/go-vela/vela-terraform/blob/a00b059b732a3768b86de12364912b44bf5e253f/cmd/vela-terraform/main.go#L307 . but, i think we're fine probably?